tools/experiments/experiments.js

```diff
@@ -348,7 +388,7 @@ function getAmpConfig() {
   xhr.send(null);
   return promise
     .then(text => {
-      const match = text.match(/self\.AMP_CONFIG=([^;]+)/);
+      const match = text.match(/self\.AMP_CONFIG=({.+?})/);
```

Comment:
For posterity, what was wrong with the previous regex?

Reply:
When we build locally with a `--config`, v0.js starts:
```javascript
self.AMP_CONFIG||(self.AMP_CONFIG={...});/*AMP_CONFIG*/
```

Whereas when we don't add a config, but rather prepend it as part of our release process, it starts:
```javascript
self.AMP_CONFIG={...};
```

The existing regex catches only the 2nd syntax, the new regex catches both. This assuming that none of the configs are objects themselves but are always strings or numbers :)

Reply:
I see. We should eventually eliminate the difference between the two workflows. #24817 was file to track this.

Meanwhile, the regex change works for now.